packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Field } from 'formik';
+import { DesktopWrapper, Dropdown, MobileWrapper, Text, SelectNative } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const TradingAssessmentDropdown = ({ item_list, onChange, values, setFieldValue, setEnableNextSection }) => {
+    React.useEffect(() => {
+        checkIfAllFieldsFilled();
+    }, [values]);
+
+    const checkIfAllFieldsFilled = () => {
+        if (values) {
+            setEnableNextSection(
+                values.cfd_experience &&
+                    values.cfd_frequency &&
+                    values.trading_experience_financial_instruments &&
+                    values.trading_frequency_financial_instruments
+            );
+        }
+    };
```

Comment:
Both `values` and `setEnableNextSection` come from the parent component. Can we just have useEffect in the parent component?

Reply:
The logic to enable the next section varies for `trading-assessment-dropdown` & `trading-assessment-radio-button`. Hence I feel it is better to handle the logic individually inside the component